### ISSUE TITLE
PrestoDB throws No nodes available to run query

### ISSUE THREAD

**Issue by leolorenzoluis** [prestodb/presto#8278]
Presto is misleading with the information that is shown. I had three connectors working properly fine such as PostgreSQL, MySQL, and Oracle. I wanted to do a benchmark test using tcph, so I started using that. A quick search on google led me to this site:

http://facebook.github.io/blog.prestodb.io/presto/tpch/screencast/2014/02/13/presto-on-aws-tpch.html

I tried to query sf1000 which ran out of memory in my cluster which is expected. However, when I tried to query against my previous queries against the 3 connectors it throwed that error. At first impression I thought it has something to do with my big query which threw an out of memory exception, so I tried to restart the server even the VM instance just to make sure no processes are hanging. 

I also did `SELECT * from system.runtime.nodes` to see the health of the nodes and everything says ACTIVE.

So basically, I try to revert my config files which includes

`datasources=jmx,tpch` and a quick google apparently says it is deprecated on the release notes in Github but wasn't able to find that in the beginning. I understand that some of the docs/videos out there are outdated but I was hoping a more meaningful error than no nodes available to run query.

Suggestion: Either spit out an error message to mention that `datasources` is deprecated, or make a clearer message.

**Comment by findepi** [comment]
I understand you got initially misled by a screencast from 2014 pointing you towards deprecated settings, but you finally found your way out. I am happy about it.
If you think there still is a bug causing presto to reject queries ("No nodes available to run query" or something), please provide some steps to reproduce.

**Comment by electrum** [comment]
Removing "datasources" is on our long list of things to do. For now it's auto configured. I'll see if we can add a warning or error if you explicitly configure it.
